### PR TITLE
Reduce warnings.

### PR DESCRIPTION
**Pull request by gwtnb**
Reduce warnings.

### CODE REVIEW

File: jubatus/util/text/json_test.cpp [hunk 1]
```diff
@@ -284,7 +284,7 @@ TEST(json, from_json)
     json j(new json_bool(false));
     bool const expected = false;
     bool const actual = json_cast<bool>(j);
-    EXPECT_EQ(expected, actual);
+    EXPECT_FALSE(actual);
```
Comment (gintenlabo) [inline jubatus/util/text/json_test.cpp]: `expected` appears unused.
How about `EXPECT_FALSE(json_cast<bool>(j));`?

Comment (gwtnb) [inline jubatus/util/text/json_test.cpp]: Thank you. I will fix it.

**Comment by kmaehashi** [comment]
Thank you!

For future reference, `EXPECT_EQ(expected, actual)` for bool types is discussed in https://code.google.com/p/googletest/issues/detail?id=322

**Comment by kmaehashi** [comment]
:+1:

**Pull request merged**